lib/lettering/font.py

```diff
@@ -144,6 +144,12 @@ def render_text(self, text, destination_group, variant=None, back_and_forth=True
 
         if self.auto_satin and len(destination_group) > 0:
             self._apply_auto_satin(destination_group, trim)
+        else:
+            # set stroke width because it is almost invisible otherwise (why?)
+            for element in destination_group.iterdescendants(SVG_PATH_TAG):
+                style = ['stroke-width:1px' if s.startswith('stroke-width') else s for s in element.get('style').split(';')]
+                style = ';'.join(style)
+                element.set('style', '%s' % style)
```

Comment:
Ah, this.  This is probably caused by the way transformations work on strokes.  The whole object is scaled _after_ drawing the stroke, so it can make the stroke look smaller or nearly invisible.  It's the same reason why the "convert line to satin column" extension isn't always right.

Your workaround may work, but it might be better to make it a _minimum_ of 1px.  Also, this is another case where we're parsing the style, which may not work for fonts with semicolons in the names.  Maybe we should use tinycss for this, too?  Perhaps add an `Element.set_style()` function?  Anyway, no rush on that, this is fine for now :)

Reply:
I didn't know fonts could have semicolons in their names. Not sure if tinycss2 could figure it out then? I thought the double point has been the problem. Maybe I am wrong though.

An Element.set_style function would be great. I came across this more often and thought something like that would be missing. Still hadn't the idea to add something like that :D